pkg/execution/state/redis_state/queue_processor.go

```diff
@@ -462,7 +459,9 @@ func (q *queue) processPartition(ctx context.Context, p *QueuePartition) error {
 		return err
 	}
 
-	eg := errgroup.Group{}
+	var processErr error
+
+ProcessLoop:
```

Comment:
why is there a `goto` statement here? I don't see anything telling the code to jump here?

Reply:
we want to break out of the loop inside the switch, so we have to label the loop for the break :(